Kattis/echo.py

```diff
@@ -0,0 +1,3 @@
+word = input().strip()
+
+print(word+" "+word+" "+word)
```

Comment:
This is a working solution. Another way is using interpolated string:
```python
print(f"{word} {word} {word}")
```
Try it out!

For reference:
https://www.programiz.com/python-programming/string-interpolation